teams/tests.py

```diff
@@ -0,0 +1,21 @@
+from .models import Teams
+import pytest
+
+
+@pytest.fixture
+def create_team():
+    team = Teams.objects.create(name='Test Team')
+    return team
```

Comment:
I would define the name as a constant like:
TEAM1 = 'Team 1' 
because if you will want to check it, it will be easier to use the constant
```suggestion
    return Teams.objects.create(name=TEAM1)
```